src/Runners/PHPUnit/ExecutableTest.php

```diff
@@ -201,6 +214,33 @@ public function run(string $binary, array $options = [], array $environmentVaria
         return $this;
     }
 
+    /**
+     * Build the full executable as we would do on the command line, e.g.
+     * php -Dzend_extension=xdebug.so vendor/bin/phpunit --teststuite suite1 --prepend xdebug-filter.php.
+     *
+     * @param $binary
+     * @param $options
+     * @param string|null $passthru
+     * @param string|null $passthruPhp
+     *
+     * @return string
+     */
+    protected function getFullCommandlineString($binary, $options, string $passthru = null, string $passthruPhp = null)
```

Comment:
Same here, I think param type needs to be `?string` for `$passthru` and `$passthruPhp`.